source/ui/BioXAS/BioXASCarbonFilterFarmView.h

```diff
@@ -0,0 +1,33 @@
+#ifndef BIOXASCARBONFILTERFARMVIEW_H
+#define BIOXASCARBONFILTERFARMVIEW_H
+
+#include <QWidget>
+#include <QLayout>
+
+#include "beamline/BioXAS/BioXASCarbonFilterFarm.h"
+
+class BioXASCarbonFilterFarmView : public QWidget
+{
+    Q_OBJECT
+
+public:
+	/// Constructor.
+	explicit BioXASCarbonFilterFarmView(BioXASCarbonFilterFarmControl *filterFarm, QWidget *parent = 0);
+	/// Destructor.
+	virtual ~BioXASCarbonFilterFarmView();
```

Comment:
For completeness, you may want to be able to return the control as well since you have a setter and signal for it already.